src/components/treeActivity/index.tsx

```diff
@@ -35,19 +35,146 @@ const EntryMessage = styled(Paragraph)`
   color: ${TEXT_GREY};
 `;
 
+const StewardshipActivityDropdownContainer = styled.div`
+  margin: -10px 0 -5px 5px;
+`;
+
+const StewardshipActivityDropdown = styled.div`
+  font-size: 15px;
+  color: ${MID_GREEN};
+`;
+
+const CenteredPagination = styled(Pagination)`
+  display: flex;
+  justify-content: center;
+  & .ant-pagination-item-active {
+    background-color: white;
+  }
+  & .ant-pagination-item-active :hover {
+    background-color: ${DARK_GREEN};
+  }
+`;
+
 interface TreeActivityProps {
   readonly stewardship: TreeCare[];
-  readonly limit?: number;
 }
 
-const TreeActivity: React.FC<TreeActivityProps> = ({ stewardship, limit }) => {
+const TreeActivity: React.FC<TreeActivityProps> = ({ stewardship }) => {
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  // convert a month from its three letter abbreviation (e.g. 'Jan') to its long form (e.g. 'January')
+  const monthShortToLong = (shortMonth: string): string => {
+    for (const month of months) {
+      if (shortMonth.substring(0, 3) === month.substring(0, 3)) {
+        return month;
+      }
+    }
+    return '';
+  };
+
+  // sort two dates by a combination of year and month
+  const sortByYearMonth = (
+    yearMonth1: { label: string },
+    yearMonth2: { label: string },
+  ) => {
+    const label1 = yearMonth1.label;
+    const label2 = yearMonth2.label;
+
+    const year1 = label1.slice(-4);
+    const month1 = months.indexOf(label1.split(' ')[0]);
+    const year2 = label2.slice(-4);
+    const month2 = months.indexOf(label2.split(' ')[0]);
+
+    if (year1 === year2 && month1 === month2) {
+      return 0;
+    } else if (year1 < year2 || (year1 === year2 && month1 < month2)) {
+      return -1;
+    } else {
+      return 1;
+    }
+  };
+
+  const [selectedMonthYear, setSelectedMonthYear] = useState(
```

Comment:
It's generally better to have state values like this in parent components and then pass the set methods as props

Reply:
@willmt80 I think it's better to have this state val in the child in this case - the `selectedMonthYear` is only relevant to the child component, which I believe means it would be better to only re-render the component vs the entire page when updating the val, as it would if it was in the parent?

Reply:
Should I put `selectedMonthYear` in the parent or child component?

Reply:
Yeah Sofie is right, leaving this in the child is good